core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -2187,14 +2187,14 @@ class KafkaApis(val requestChannel: RequestChannel,
     authorizer match {
       case None =>
         sendResponseMaybeThrottle(request, requestThrottleMs =>
-          new DescribeAclsResponse(requestThrottleMs,
+          DescribeAclsResponse.prepareResponse(requestThrottleMs,
             new ApiError(Errors.SECURITY_DISABLED, "No Authorizer is configured on the broker"), util.Collections.emptySet()))
       case Some(auth) =>
         val filter = describeAclsRequest.filter
         val returnedAcls = new util.HashSet[AclBinding]()
         auth.acls(filter).asScala.foreach(returnedAcls.add)
```

Comment:
This is existing code, but is there a reason why we can't just do `asJava` instead of this?